src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts

```diff
@@ -445,10 +445,13 @@ class indexes extends viewModelBase {
         }
     }
 
-    createIndexesUrlObservableForNode(nodeTag: string) {
+    createIndexesUrlObservableForNode(nodeTag: string, indexProgress: indexProgress) {
         return ko.pureComputed(() => {
+            const name = indexProgress.name;
+            const indexName = name.startsWith("replacementof/") ? name.replace("replacementof/", "") : name;
```

Comment:
https://github.com/ravendb/ravendb/blob/4f0255da54ea1c09df3026d8ad0d71a0b80bf12b/src/Raven.Studio/typescript/models/database/index/index.ts#L7

here we use different casing. In other place we removed toLowerCase. 

It is quite suspicious for me. 

Did you check that?

Reply:
right - fixed !!